modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -161,126 +161,9 @@ rosa create cluster --sts
-+
-[NOTE]
-====
-If you are installing into a VPC, the *Machine CIDR* range must match the VPC subnets.
-====
-+
-[IMPORTANT]
-====
-CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
-====
-
-. Under the *Cluster roles and policies* page, select *Auto* mode. With this mode, you can automatically create the cluster-specific Operator IAM roles and OIDC provider.
-+
-[NOTE]
-====
-To enable *Auto* mode, the {cluster-manager} IAM role must have administrator capabilities.
-
-If you alternatively want to create the cluster-specific IAM roles and the OIDC provider by using *Manual* mode, see _Creating a cluster using customizations_.
-====
-
-. Optional: Specify a *Custom operator roles prefix* for your cluster-specific Operator roles. 
-+
-[NOTE]
-====
-By default, the cluster-specific Operator role names are prefixed with the cluster name and random 4-digit hash. You can optionally specify a custom prefix to replace `<cluster_name>-<hash>` in the role names. The prefix is applied when you create the cluster-specific Operator IAM roles. For information about the prefix, see _About custom Operator IAM role prefixes_.
-====
-
-. Select *Next*.
-
-. On the *Cluster update strategy* page, configure your update preferences:
-.. Choose a cluster update method:
-** Select *Individual updates* if you want to schedule each update individually. This is the default option.
-** Select *Recurring updates* to update your cluster on your preferred day and start time, when updates are available.
-+
-[IMPORTANT]
-====
-Even when you opt for recurring updates, you must update the account-wide and cluster-specific IAM resources before you upgrade your cluster between minor releases.
-====
-+
-[NOTE]
-====
-You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
-====
-+
-.. If you opted for recurring updates, select a preferred day of the week and upgrade start time in UTC from the drop-down menus.
-.. Optional: You can set a grace period for *Node draining* during cluster upgrades. A *1 hour* grace period is set by default.
-.. Click *Next*.
-+
-[NOTE]
-====
-In the event of critical security concerns that significantly impact the security or stability of a cluster, Red Hat Site Reliability Engineering (SRE) might schedule automatic updates to the latest z-stream version that is not impacted. The updates are applied within 48 hours after customer notifications are provided. For a description of the critical impact security rating, see link:https://access.redhat.com/security/updates/classification[Understanding Red Hat security ratings].
-====
+. To deploy a cluster quickly, select the default options and select *Next* on each page until you reach the *Review your ROSA cluster* page.
```

Comment:
I could follow this explainer, but I do feel it may be a little sparse. There are five subsequent screens requiring you to click "Next" to get to the Review your ROSA cluster details. I know specifying would create a manual update requirement for any UI changes though, but maybe giving the users some guideposts may help. We could even list the pages to accept the defaults for:

![image](https://user-images.githubusercontent.com/16167833/185399702-20246364-de02-4395-96b1-55e442384f5b.png)

Reply:
Great suggestion, thanks. I have updated the line to clarify which pages the step relates to.